gremlin-python/src/main/jython/gremlin_python/driver/serializer.py

```diff
@@ -53,6 +53,22 @@ def eval(self, args):
         return args
 
 
+class Session(Processor):
+
+    def authentication(self, args):
+        return args
+
+    def eval(self, args):
+        return args
+
+    def close(self, args):
+        # close session, 'gremlin' in args as tips and not be executed actually
+        gremlin = args.get('gremlin')
+        if not gremlin:
+            args['gremlin'] = 'session.close()'
```

Comment:
As in the other languages, I'd prefer we not include any extra arguments to the "close" message. Please modify this python PR to match what was done on the other language PRs.

Reply:
en. I will remove this like PR in .Net